### ISSUE TITLE
train baseline model on 6 instruction set

### ISSUE THREAD

**Issue by ontocord** [ontocord/MDEL#42]
We need to eval the experts that are merged against if we trained a 1b model all together.
(2) Can we get one pythia 1b trained with all layers on the 6 dataset we have, and 
(1) then 1b trained with just the upper layers. 

To keep it fair, we would need to get the exact same 1000 random train example for the 6 dataset we used in the ohter experiments. And we merge the 6 experts with basic averaging and run the same eval from the 6 dataset on that model. 

This will give us a comparison of (1) training all layers on same token and data, (2) training some layers on same token and daa, and (3) merging with different experts trained on same compute.

**Comment by mrseeker** [comment]
Have you tried using the EleutherAI eval harness? It should give you a nice representation on how well the model performs, and can be used as an indicator?

**Comment by mrcabbage972** [comment]
I didn't understand the part about the 1000 training examples. Our datasets are much bigger than that!

**Comment by ontocord** [comment]
didn't we just train our models on 1000 examples only? Or did i misunderstand that

**Comment by ontocord** [comment]
We definately should try on eleuther eval harness. but just testing validity loss will tell us something too. regular finetuing vs. expert finetuning + merge

**Comment by mrcabbage972** [comment]
We have an issue for Eval Harness in the backlog.

**Comment by ontocord** [comment]
so i am told that:
It seems they were trained on 1k batches
I think the batch size was 8 because of the number of GPUs
So that gives uhs 8k samples

So the above 1000 examples should be 8K examples.

**Comment by jordiclive** [comment]
@ontocord  for 2. we want layer_9,10,11,12,13 ?

**Comment by mrcabbage972** [comment]
@jordiclive @ontocord 
We had used layers 9-13 when we trained the experts. See: https://github.com/ontocord/MDEL/blob/main/src/mdel/train.sh#L4

**Comment by mrcabbage972** [comment]
@jordiclive Any updates on this issue?

**Comment by jordiclive** [comment]
@mrcabbage972 I trained 1. a model (all layers) on the exact splits...https://wandb.ai/ontocord/jordi_testing/runs/hu8j9ta1?workspace=user-jordanclive if you toggle the evaluation.

But I  then thought we decided on automating the experiment again with more training data/less validation, maybe same amount of final testing data #47
